src/cljs/main/broadfcui/common/notifications.cljs

```diff
@@ -115,154 +115,56 @@
-                  "#" div-id " .markdown-body li:before {content: counters(item, \".\") \".\";\n"
-                  "  counter-increment: item; margin-left: -2em; position: absolute}\n"
-                  "#" div-id " .markdown-body li li:before {margin-left: -3em}\n"
-                  "#" div-id " .markdown-body li li li:before {margin-left: -4em}\n"
-                  "#" div-id " .markdown-body li {display: block}")]
-            (if-not page-2?
-              [markdown/MarkdownView {:text broad}]
-              [:div {}
-               [:div {:style {:fontSize "80%"}}
-                [markdown/MarkdownView {:text onix}]]
-               [:div {:style {:padding "1rem" :marginTop "0.5rem"
-                              :border style/standard-line :background (:background-light style/colors)}}
-                [:label {:style {:marginBottom "0.5rem" :display "block"}}
-                 [:input {:type "checkbox"
-                          :onChange #(swap! state assoc :terms-agreed? (.. % -target -checked))
-                          :data-test-id "agree-terms"}]
-                 "I agree to the terms of this Agreement."]
-                [:label {:style {:display "block"}}
-                 [:input {:type "checkbox"
-                          :onChange #(swap! state assoc :cloud-terms-agreed? (.. % -target -checked))
-                          :data-test-id "agree-cloud-terms"}]
-                 "I agree to the Google Cloud Terms of Service."]
-                [:div {:style {:paddingLeft "1rem"}} "Google Cloud Terms of Service: "
-                 (links/create-external {:href "https://cloud.google.com/terms/"}
-                   "https://cloud.google.com/terms/")]]])])
-         :data-test-id "eula-modal"
-         :button-bar (flex/box
-                       {:style {:justifyContent "center" :width "100%"}}
-                       (when page-2?
-                         [buttons/Button
-                          {:text "Back"
-                           :style {:marginRight "2rem"}
-                           :onClick #(swap! state dissoc :page-2? :terms-agreed? :cloud-terms-agreed?)}])
-                       [buttons/Button
-                        (if-not page-2?
-                          {:text "Review Terms of Service"
-                           :data-test-id "review-terms-of-service"
-                           :onClick #(swap! state assoc :page-2? true)}
-                          {:text "Accept"
-                           :data-test-id "accept-terms-of-service"
-                           :disabled? (when-not (and terms-agreed? cloud-terms-agreed?)
-                                        "You must check the boxes to accept the agreement.")
-                           :onClick accept-eula})])}]))})
+    (fn [{:keys [state this]}]
+      (let [{:keys [dismissed?]} @state]
+        (when (not dismissed?)
+          [:div {}
+            [:div {:style {:height "66px"
```

Comment:
Indentation looks a little funky, key on next line should be aligned with this key, not the brace. Also, `px` values should be able to be just specified as integers.